stable/cerebro/Chart.yaml

```diff
@@ -6,7 +6,7 @@ sources:
  - https://github.com/yannart/docker-cerebro
```

Comment:
this should probably be removed if using another image

Reply:
Good point, will do.